bot/constants.py

```diff
@@ -13,5 +12,13 @@
 LOG_FILE = pathlib.Path("log/gurkbot.log")
 
 
+class Emojis(NamedTuple):
+    issue = "<:IssueOpen:794119024367632405>"
+    issue_closed = "<:IssueClosed:794118652219359253>"
+    pull_request = "<:PROpen:794118652014231562>"
+    pull_request_closed = "<:PRClosed:794120818908463134>"
+    merge = "<:PRMerged:794119023687761941>"
```

Comment:
```suggestion
class Emojis(NamedTuple):
    issue_emoji = "<:IssueOpen:794119024367632405>"
    issue_closed_emoji = "<:IssueClosed:794118652219359253>"
    pull_request_emoji = "<:PROpen:794118652014231562>"
    pull_request_closed_emoji = "<:PRClosed:794120818908463134>"
    merge_emoji = "<:PRMerged:794119023687761941>"
```
Please suffix `emoji` to the class variable name as it will be required once we start using [Smart Config](https://github.com/python-discord/smartconfig).